src/services/staffService.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ *  staffService.ts
+ *
+ *  @copyright 2024 Digital Aid Seattle
+ *
+ */
+
+import { PostgrestSingleResponse } from "@supabase/supabase-js";
+import { supabaseClient } from "./supabaseClient";
+import { v4 as uuid } from 'uuid';
+import { read, utils } from "xlsx";
+
+type Staff = {
+    id: string,
+    created_at: Date,
+    name: string,
+    email: string,
+    roles: string[],
+}
+
+class StaffService {
+    async getStaff(): Promise<Staff[]> {
+        return supabaseClient.from('staff')
+            .select()
+            .then((res: PostgrestSingleResponse<Staff[]>) => {
+                if (res.error) {
+                    throw new Error(res.error.message)
+                }
+                return res.data as Staff[]
+            })
+    }
+
+    async postStaff(rows: Staff[]) {
+        const { error } = await supabaseClient
+            .from('staff')
+            .insert(rows)
+            .select()
+        if (error) {
+            throw new Error(error.message);
+        }
+    }
+
+    // referenced example at https://docs.sheetjs.com/docs/demos/frontend/react/
+    async handleParse(file: File) {
```

Comment:
minor: add return type

I think it would be like:
    async handleParse(file:File) : Promise<Staff[]> {